src/components/Template/Template.mdx

```diff
@@ -5,496 +5,123 @@ import { changelogData } from "./templateChangelogData";
-    <TemplateAboveHeader>
-      // ...
-    </TemplateAboveHeader>
-    <TemplateHeader>
-      // ...
-    </TemplateHeader>
-  </TemplateBreakout>
-  <TemplateContent>
-    <TemplateContentTop>
-      // ...
-    </TemplateContentTop>
-  </TemplateContent>
-</Template>
-`}
-  language="jsx"
-/>
-
-<Description of={TemplateContentTop} />
-
-#### TemplateContentPrimary
-
-<Source
-  code={`
-<Template>
-  <TemplateBreakout>
-    <TemplateAboveHeader>
-      // ...
-    </TemplateAboveHeader>
-    <TemplateHeader>
-      // ...
-    </TemplateHeader>
-  </TemplateBreakout>
-  <TemplateContent>
-    <TemplateContentPrimary>
-      // ...
-    </TemplateContentPrimary>
-  </TemplateContent>
-<Template>
-`}
-  language="jsx"
-/>
-
-<Description of={TemplateContentPrimary} />
-
-#### TemplateContentSidebar
-
-<Source
-  code={`
-<Template>
-  <TemplateBreakout>
-    <TemplateAboveHeader>
-      // ...
-    </TemplateAboveHeader>
-    <TemplateHeader>
-      // ...
-    </TemplateHeader>
-  <TemplateBreakout>
-  <TemplateContent sidebar="right">
-    <TemplateContentPrimary>
-      // ...
-    </TemplateContentPrimary>
-    <TemplateContentSidebar>
-      // RIGHT SIDEBAR
-    </TemplateContentSidebar>
-  </TemplateContent>
-</Template>
+This means you may use `TemplateMain` or `TemplateMainNarrow` depending on your needs,
+but the two should not be used in conjunction.
 
-<Template>
-  <TemplateBreakout>
-    <TemplateAboveHeader>
-      // ...
-    </TemplateAboveHeader>
-    <TemplateHeader>
-      // ...
-    </TemplateHeader>
-  </TemplateBreakout>
-  <TemplateContent sidebar="left">
-    <TemplateContentSidebar>
-      // LEFT SIDEBAR
-    </TemplateContentSidebar>
-    <TemplateContentPrimary>
-      // ...
-    </TemplateContentPrimary>
-  </TemplateContent>
-<Template>
-`} language="jsx" />
+The `TemplateMain` and `TemplateMainNarrow` components render the content as a
+`<main>` element with a default "id" of `"mainContent"`. This should be used as
+the anchor element that the skip navigation link points to. If your application
+is using the current Header, it already contains the skip navigation feature but
+make sure it is pointing to the correct anchor element.
 
-<Description of={TemplateContentSidebar} />
+Resources
 
-#### TemplateContentBottom
+- [WebAim Skip Navigation Link](https://webaim.org/techniques/skipnav/)
+- [A11ymatters Skip Navigation Link](https://www.a11ymatters.com/pattern/skip-link/)
 
-<Source
-  code={`
-<Template>
-  <TemplateBreakout>
-    <TemplateAboveHeader>
-      // ...
-    </TemplateAboveHeader>
-    <TemplateHeader>
-      // ...
-    </TemplateHeader>
-  </TemplateBreakout>
-  <TemplateContent>
-    <TemplateContentTop>
-      // ...
-    </TemplateContentTop>
-    <TemplateContentPrimary>
-      // ...
-    </TemplateContentPrimary>
-    <TemplateContentBottom>
-      // ...
-    </TemplateContentBottom>
-  </TemplateContent>
-</Template>
-`}
-  language="jsx"
-/>
+## Template Sidebar Configuration
```

Comment:
We should either expand this section to detail the placement and function of all of the grid regions or we should add a section for each region. I am leaning toward adding separate sections to describe each region.

Reply:
Okay, I think that makes sense! Will leave this work for the documentation ticket.